CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### ✅ Added
 - `ChannelListQuery.membersLimit` param for controlling the number of members returned for each channel [#1721](https://github.com/GetStream/stream-chat-swift/issues/1721)
+- Extra data for message can now be passed directly from `ComposerVC`. In order to pass the extra data for all messages, override the `publishMessage(sender:)` function and adjust the `content.extraData` prior calling `super.publishMessage(sender:)` [#1722](https://github.com/GetStream/stream-chat-swift/pull/1722)
```

Comment:
nit: let's keep it short? 🙂
```suggestion
- Make it possible to change extra data when editing the message [#1722](https://github.com/GetStream/stream-chat-swift/pull/1722)
```

Reply:
The `when editing the message` part is misleading. But Ill cut it short :)